src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -216,6 +216,41 @@ def forward(self, input_ids, past_key_values_length=0, position_ids=None):
             return self.weight[position_ids]
 
 
+def eager_attn_forward(
```

Comment:
```suggestion
def eager_attention_forward(
```
(consistent with other implementations)